src/metrics/listener.ts

```diff
@@ -143,6 +144,14 @@ export class MetricsListener {
 
     const dist = new Distribution(name, [{ timestamp: metricTime, value }], ...tags);
 
+    if (!this.apiKey) {
+      const errorMessage = "api key not configured, see https://dtdg.co/sls-node-metrics";
+      if (this.config.logForwarding || this.isAgentRunning) {
+        logDebug(errorMessage);
+      } else {
+        logError(errorMessage);
+      }
```

Comment:
The condition is unnecessary because of https://github.com/DataDog/datadog-lambda-js/blob/dc6e1109f576f20e015cd37c4deff3f14d66a024/src/metrics/listener.ts#L136 and https://github.com/DataDog/datadog-lambda-js/blob/dc6e1109f576f20e015cd37c4deff3f14d66a024/src/metrics/listener.ts#L140. 
```suggestion
        logError(errorMessage);
        return;
```